azure_functions_worker/dispatcher.py

```diff
@@ -288,6 +288,9 @@ async def _handle__worker_init_request(self, request):
         if not DependencyManager.is_in_linux_consumption():
             DependencyManager.prioritize_customer_dependencies()
 
+        if DependencyManager.is_in_linux_consumption():
```

Comment:
is the linux consumption check needed?

Reply:
Right now, we are just minimizing the scope of this change. For the first iteration we are only doing this for consumption. Will test and update for dedicated in the next iteration if needed.